src/html/fetch-content.js

```diff
@@ -87,6 +87,12 @@ async function fetchContent(context, {
       sourceLocation,
       sourceHash,
     };
+
+    const lastModified = res.headers.get('last-modified');
+    logger.info(`last-modified: ${lastModified}`);
+    if (lastModified) {
+      context.content.data = { ...context.content.data, lastModified };
```

Comment:
why not: `context.content.data.lastModified = lastmodified` ? this avoids creating a new object.